### DISCUSSION TITLE
Make the checkmarks' click area bigger on desktop

### DISCUSSION THREAD

**Discussion by kawazoe** [lukevella/rallly#156] [Ideas]
Me and my friend often use rallly to plan stuff a month at a time. With the current touch target for confirming dates, it can be difficult to click on 30+ days in a row, even with a mouse. High precision repeated tasks like this can cause strains over time. Ideally, the entire cell should toggle the checkbox, just like on mobile.

**Comment by lukevella** [comment]
Yeh it's been on my mind as well. Definitely could be bigger. I'll be addressing this soon.
